src/components/Todos/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Dispatch, FormEvent, SetStateAction, useEffect, useRef,
+} from 'react';
+import { createTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  userId: number;
+  todos: Todo[];
+  setTodos: Dispatch<SetStateAction<Todo[]>>;
+  setError: Dispatch<SetStateAction<boolean>>;
+  setErrorMessage: Dispatch<SetStateAction<string>>;
+  setLoader: Dispatch<SetStateAction<boolean>>;
+  title: string;
+  setTitle: Dispatch<SetStateAction<string>>;
+};
+
+export const Header: React.FC<Props> = ({
+  userId,
+  todos,
+  setTodos,
+  setError,
+  setErrorMessage,
+  setLoader,
+  title,
+  setTitle,
+}) => {
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    // focus the element with `ref={newTodoField}`
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
+
+  const handleSumbit = (event: FormEvent) => {
+    event.preventDefault();
+    const normalizedTitle = title.trim();
+
+    if (title.length < 1) {
```

Comment:
```suggestion
    if (!normalizedTitle.length) {
```

i still can add empty todo

Reply:
yes, but in the working version we can also add empty Todos, however, if we add a name at the beginning of which there are spaces, these spaces are removed. I repeated the functionality from the working version.